presto-kudu/bin/run_kudu_tests.sh

```diff
@@ -13,24 +13,14 @@
 
 set -euo pipefail -x
 
-export KUDU_MASTER_RPC_PORT=17051
-export DOCKER_HOST_IP=$(docker network inspect bridge --format='{{index .IPAM.Config 0 "Gateway"}}')
```

Comment:
can we still use `docker network inspect` instead of hardcoding IP addresses?

Is it possible to expose kudu master ports to host machine like as `17051, 17052, 17053` and use them with `localhost:17051,localhost:17052,localhost:17053`? Then also we wouldn't care about IP addresses of docker containers.

Reply:
done